### PR TITLE
Removing Netbox v3 client

### PR DESCRIPTION
**Pull request by johannwagner**
I deleted some tests, which are not necessary anymore.
They were introduced temporary while implementing a Netbox v4 client.

### CODE REVIEW

File: cosmo/tests/test_netboxclient.py [hunk 1]
```diff
@@ -16,46 +16,16 @@
...
+
     responseData = nc.get_data(TEST_DEVICE_CFG)
     assert responseData == mockAnswer
-    versionDetectMock.assert_called_once()
-    dataMock.assert_called_once()
-    kwargs = dataMock.call_args.kwargs
-    assert 'json' in kwargs
-    assert 'query' in kwargs['json']
-    ncQueryStr = kwargs['json']['query']
-    for device in [*TEST_DEVICE_CFG['router'], *TEST_DEVICE_CFG['switch']]:
```
Comment (loulecrivain) [inline cosmo/tests/test_netboxclient.py:60]: why delete this test data?

File: cosmo/tests/integration_test.py [hunk 2]
```diff
@@ -23,37 +23,37 @@ def test_missing_netbox_api_token(mocker):
...
     testEnv.stop()
     assert os.path.isfile('machines/test0001/generated-cosmo.json')
 
 def test_device_processing_error(mocker):
     testEnv = utils.CommonSetup(mocker, cfgFile='cosmo/tests/cosmo.devgen_nix.yml')
     with open(f"cosmo/tests/test_case_vendor_unknown.yaml") as f:
-        utils.RequestResponseMock.patchTool(
-            mocker,{'status_code': 200, 'text': '{"data": ' + json.dumps(yaml.safe_load(f)) + '}'})
+        test_data = yaml.safe_load(f)
+        utils.RequestResponseMock.patchNetboxClient(mocker, **test_data)
```
Comment (loulecrivain) [inline cosmo/tests/integration_test.py:56]: I like it, cleaner and more concise :+1:

File: cosmo/tests/test_netboxclient.py (hunk 1 above)
Comment (johannwagner) [reply cosmo/tests/test_netboxclient.py:60]: The different mocks are now called multiple times with different parameters and its not very good to test. We at least could test, that they are called.

Comment (johannwagner) [reply cosmo/tests/test_netboxclient.py:60]: Uh, thats not true for this part.

**Pull request merged**